packages/map-template/src/components/Map/Map.jsx

```diff
@@ -1,25 +1,27 @@
 import { useEffect } from "react";
 import { useRecoilState, useRecoilValue } from 'recoil';
-import { mapTypes } from "../../constants/mapTypes";
-import useLiveData from '../../hooks/useLivedata';
-import venuesState from '../../atoms/venuesState';
-import GoogleMapsMap from "./GoogleMapsMap/GoogleMapsMap";
-import MapboxMap from "./MapboxMap/MapboxMap";
-import mapsIndoorsInstanceState from '../../atoms/mapsIndoorsInstanceState';
-import userPositionState from '../../atoms/userPositionState';
-import directionsServiceState from '../../atoms/directionsServiceState';
-import mapTypeState from '../../atoms/mapTypeState';
-import currentVenueNameState from '../../atoms/currentVenueNameState';
 import apiKeyState from '../../atoms/apiKeyState';
+import bearingState from '../../atoms/bearingState';
+import currentVenueNameState from '../../atoms/currentVenueNameState';
+import directionsServiceState from '../../atoms/directionsServiceState';
+import filteredLocationsByExternalIDState from '../../atoms/filteredLocationsByExternalIDState';
+import filteredLocationsState from '../../atoms/filteredLocationsState';
 import gmApiKeyState from '../../atoms/gmApiKeyState';
+import locationIdState from '../../atoms/locationIdState';
 import mapboxAccessTokenState from '../../atoms/mapboxAccessTokenState';
-import filteredLocationsState from '../../atoms/filteredLocationsState';
-import filteredLocationsByExternalIDState from '../../atoms/filteredLocationsByExternalIDState';
-import tileStyleState from '../../atoms/tileStyleState';
-import startZoomLevelState from '../../atoms/startZoomLevelState';
+import mapsIndoorsInstanceState from '../../atoms/mapsIndoorsInstanceState';
+import mapTypeState from '../../atoms/mapTypeState';
+import pitchState from '../../atoms/pitchState';
 import positionControlState from '../../atoms/positionControlState';
-import locationIdState from '../../atoms/locationIdState';
+import startZoomLevelState from '../../atoms/startZoomLevelState';
+import tileStyleState from '../../atoms/tileStyleState';
+import userPositionState from '../../atoms/userPositionState';
+import venuesState from '../../atoms/venuesState';
+import { mapTypes } from "../../constants/mapTypes";
 import setMapZoomLevel from "../../helpers/SetMapZoomLevel";
+import useLiveData from '../../hooks/useLivedata';
+import GoogleMapsMap from "./GoogleMapsMap/GoogleMapsMap";
+import MapboxMap from "./MapboxMap/MapboxMap";
```

Comment:
Did you manage to revert the import changes? 🙏 @rocu-mapspeople

Reply:
will check tomorrow @andreeaceachir142 and compare with the current main branch, and update again if it's not consistent.

Reply:
Great!